logbook-core/src/main/java/org/zalando/logbook/Multimap.java

```diff
@@ -0,0 +1,89 @@
+package org.zalando.logbook;
+
+/*
+ * #%L
+ * Logbook: Core
+ * %%
+ * Copyright (C) 2015 - 2016 Zalando SE
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import java.util.*;
+
+import static java.util.stream.Collectors.toSet;
+
+/**
+ * Created by clalleme on 02/04/2016.
+ */
+public interface Multimap<K, V> extends Map<K, Collection<V>> {
```

Comment:
Note that my intend is not to rewrite guava here. All tested have been passed while returning null.
if we need this type of feature we can user Map.getOrDefault
Anyway in BasicMultimap implementation I overrode 'get' in order to mimic the guava feature.